src/agent/rustjail/src/cgroups/systemd/manager.rs

```diff
@@ -41,7 +41,7 @@ pub struct Manager {
 impl CgroupManager for Manager {
     fn apply(&self, pid: pid_t) -> Result<()> {
         let unit_name = self.unit_name.as_str();
-        if self.dbus_client.unit_exist(unit_name).unwrap() {
```

Comment:
> It will be useful to add context here to include the unit_name in the error message.

What about having the code that actually hits the failure to provide a useful context ? e.g. complete
path name and action/syscall being performed ?

Reply:
Will add that

Reply:
@amshinde Thanks for the suggestion. Added some context at various points in this PR.

Reply:
@gkurz Can you give it a whirl again, see if that can narrow down where we fail?